src/common/interfaces/Events.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Enum representing the various custom events used in the application.
+ *
+ * The string value MUST be lowercase.
+ */
+const enum Events {
```

Comment:
Any chance we could put this in src/enum/Event.ts ?

Reply:
I merged with the latest version of master and then moved the enum into a new enums folder.